docs/operator-docs/run-a-stress-test.md

```diff
@@ -0,0 +1,51 @@
+---
+id: run-a-stress-test
+title: Run A Stress Test
+hide_title: true
+---
+
+# Run A Stress Test
+
+## UAT Stress Test
+
+1. Login CloudTamer
+
+2. Select the Project
+
+3. Generate AWS Long Term Access Key
+
+* For steps on creating an AWS Access Key go to [AWS Long Term Access Key](../operator-docs/aws-long-term-access-key).
+* Copy the AWS Long Term Access Key.
+
+4. AWS CloudFront
+
+5. Run Local Test
+
+* In your terminal, enter the following:
+
+```bash
+
+git clone <your test repository git URL>
+git checkout develop
+git status
+
+```
+
+* Then run CNMTestHarness log file. Check with your developer on where to find your python resource for this test.
```

Comment:
Interesting - what is this?

Reply:
Let me look at my reference documents again to validate.